src/runtime_src/core/edge/hw_em/CMakeLists.txt

```diff
@@ -14,6 +17,12 @@ file(GLOB EM_SRC_FILES
   "${COMMON_EM_SRC_DIR}/*.cpp"
   "${EM_SRC_DIR}/*.h"
   "${EM_SRC_DIR}/*.cpp"
+  "${COMMON_SRC_DIR}/common/*.h"
+  "${COMMON_SRC_DIR}/common/*.cpp"
+  "${COMMON_SRC_DIR}/../common/system.h"
+  "${COMMON_SRC_DIR}/../common/system.cpp"
+  "${COMMON_SRC_DIR}/../common/device.cpp"
+  "${COMMON_SRC_DIR}/../common/device.h"
```

Comment:
Why use file name? Use wildcard is better. Isn't it?
"${COMMON_SRC_DIR}/../common/*.h"
"${COMMON_SRC_DIR}/../common/*.cpp"

Reply:
there are many other files in 'common' folder but we are using specific files only.

Reply:
I'm good with this change.